modules/core/src/main/java/org/apache/ignite/internal/processors/service/LazyServiceConfiguration.java

```diff
@@ -36,6 +37,10 @@ public class LazyServiceConfiguration extends ServiceConfiguration {
     @GridToStringExclude
     private transient Service srvc;
 
+    /** Node filter. */
+    @GridToStringExclude
+    private transient IgnitePredicate<ClusterNode> nodeFilter;
```

Comment:
Suggestion: let's emphasize that it is a protection of desacralization failure on base `nodeFilter` until `nodeFilterBytes` is attempted to read.

Reply:
It's already emphasized by java key word - transient. Isn't it enough?

Reply:
I think comment is much better.